common/src/main/java/com/wynntils/features/user/CombatXPGainMessageFeature.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.features.user;
+
+import com.wynntils.core.config.Config;
+import com.wynntils.core.features.UserFeature;
+import com.wynntils.core.notifications.NotificationManager;
+import com.wynntils.mc.event.SetExperienceEvent;
+import com.wynntils.wynn.event.WorldStateEvent;
+import com.wynntils.wynn.model.CharacterManager;
+import com.wynntils.wynn.utils.WynnUtils;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+
+public class CombatXPGainMessageFeature extends UserFeature {
```

Comment:
I think it should be `CombatXpGainMessageFeature`, and similarly for the variables below, `newTickXp`etc. Combining acronyms traditionally written in all-caps with CamelCase is always difficult, but the normal Java convention is to treat them as words and camel-case them ("Url"), and I think that's the best. That is also what we have done before wrt to XP.